package.json

```diff
@@ -9,12 +9,15 @@
     "./cypress/**/*.*"
   ],
   "resolutions": {
+    "@types/react": "18.2.20",
```

Comment:
where does the requirement come from for all these resolutions?

Reply:
Looks like changing to Next 13.x introduces:
```
GHSA-ww39-953v-wcq6|next>watchpack>chokidar>glob-parent
GHSA-ww39-953v-wcq6|next>watchpack>watchpack-chokidar2>chokidar>glob-parent
```

So pinning the version of `glob-parent` to at least `5.1.2` seems to be required.